common/app-starters-security-common/README.adoc

```diff
@@ -0,0 +1,24 @@
+=== `App Starters Security` Common Module
+
+This artifact contains a Spring Boot auto-configuration to disable security in the application that set
+`spring.cloud.security.enabled = false` (default).
```

Comment:
You are still missing `stream` in the middle...

Also consider to follow "One sentence per line rule": https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line